libexec/scoop-shim.ps1

```diff
@@ -0,0 +1,241 @@
+# Usage: scoop shim <subcommand> [<shim_names>] [<command_path> [<args>...]] [-g(lobal)]
+# Summary: Manipulate Scoop shims
+# Help: Manipulate Scoop shims: add, remove/rm, list, info, alter, etc.
```

Comment:
```suggestion
# Help: Manipulate Scoop shims: add, rm, list, info, alter, etc.
```

Reply:
Why not add `remove` and use `rm` as alias?

Reply:
I commented below. Basically:

> For consistency with other scoop commands

Reply:
Oh, I've not noticed. And, why not add `remove` to all other command? It's strange that only `rm` make sense.

Alias could be added in a smarter way, so I'll remove it first and add `remove` to all scoop command in another PR.

Reply:
`remove` is longer to type 🤣

`rm` has been used since long time because of unix-like tools, it is synonymous with remove

Reply:
Yes, even myself never type `remove` 😄